relay-server/src/actors/processor.rs

```diff
@@ -526,14 +526,14 @@ impl FromMessage<RateLimitFlushBuckets> for EnvelopeProcessor {
 #[derive(Clone)]
 pub struct EnvelopeProcessorService {
     global_config: Arc<GlobalConfig>,
+    global_config_receiver: Receiver<Arc<GlobalConfig>>,
```

Comment:
it feels a bit ugly to have both of thes, although if we simply remove the global_config, we might end up having different global_configs for different parts of the processing.